InfrastructureManager/system_manager.py

```diff
@@ -160,6 +174,38 @@ def get_swap_usage(self, secret):
 
     return json.dumps(swap_stats_dict)
 
+  def get_loadavg(self, secret):
+    """ Returns info from /proc/loadavg.
+    See `man proc` for more details.
+
+    Args:
+      secret: The secret of the deployment; used for authentication.
+    Returns:
+      A dictionary containing average load for last 1, 5 and 15 minutes,
+      and information about running and scheduled entities,
+      and PID of the most recently added process.
+    """
+    if self.secret != secret:
+      return self.__generate_response(False,
+        InfrastructureManager.REASON_BAD_SECRET)
+
+    loadavg = open("/proc/loadavg").read().split()
```

Comment:
I would recommend using a with statement here to ensure the descriptor is closed.